src/Components/ToggleButton.tsx

```diff
@@ -4,21 +4,35 @@ type ToggleButtonProps = {
   label: string;
   active: boolean;
   onClick: () => void;
+  activeColor?: string;
 };
 
-const ToggleButton: React.FC<ToggleButtonProps> = ({ label, active, onClick }) => {
+const ToggleButton: React.FC<ToggleButtonProps> = ({ 
+  label,
+  active,
+  onClick,
+  activeColor = 'rgb(141, 194, 199)',
+}) => {
+  const inactiveColor = 'rgb(216, 217, 204)';
+  const textColor = 'rgb(60, 64, 63)';
   return (
     <button
       onClick={onClick}
       className='togglable'
       style={{
-        backgroundColor: active ? 'rgb(141, 194, 199)' : 'rgb(216, 217, 204)',
-        color: 'rgb(60, 64, 63)',
+        backgroundColor: active ? activeColor : inactiveColor, 
+        color: textColor, 
       }}
     >
       {label}
     </button>
   );
 };
 
+const ColorsToggleButton = {
+  palePink: 'rgb(246, 170, 147)',
+  purple: 'rgb(161, 122, 211)',
+};
```

Comment:
Might be good to not have these in here, having another files and naming them based on the project, e.g. `const FoodChainMagnatePalette` and naming the keys similarly themed: e.g. from `palePink` to `discounters`, and `purple` to `cfo`.

Reply:
This is now fixed - I also added default colours to this palette so RGB values only appear in the palette and the CSS.